NEWS.md

```diff
@@ -122,6 +122,8 @@
 
 16. `rbind` and `rbindlist` of an item containing an ordered factor with levels containing an `NA` (as opposed to an NA integer) could segfault, [#3601](https://github.com/Rdatatable/data.table/issues/3601). This was a a regression in v1.12.2. Thanks to Damian Betebenner for reporting.
 
+17. `rbind.data.frame` on `IDate` columns corrupted the `integer` storage mode, [#2008](https://github.com/Rdatatable/data.table/issues/2008). Thanks to @rmcgehee for reporting.
```

Comment:
'corrupted' implies something broken (seriously!) but it's just a type change, still worked.  I'll soften it when merging.